k8s/nginx/schema.yaml

```diff
@@ -24,6 +24,11 @@ properties:
     description: The number of Pods run within NGINX solution
     default: 3
     minimum: 1
+  ingress.available:
```

Comment:
In case of Nginx the variable is actually used to determine if obtaining a public IP is possible - I think the variable name similar to `publicIP.available` or `publicEndpoint.available` would make it more readable. I know the variable type would need to be the same (`INGRESS_AVAILABLE`), but it is like abstracting the design from implementation - the goal of this variable is to indicate if public connectivity is available, not Ingress in particular.

Reply:
Done :+1: